recipes/metric_learn/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "metric-learn" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: fd43bf76aa2f14dfaa40cdbacb3b2fbab25481dafa85491c5cf49ecabf94dab9
+
+build:
+  number: 0
```

Comment:
I think this can probably be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I guess so, thanks!